src/test/java/life/qbic/subscriptions/SubscriptionControllerTest.java

```diff
@@ -85,21 +84,15 @@ void whenValidInputIsProvidedPostSubscriptionsTokensRespondsCreated() throws Exc
         .andExpect(content().string(encrypted));
   }
 
-  @Disabled("Disabled until authorization is tackled")
   @Test
-  @DisplayName("When authorization is missing, DELETE /subscriptions responds UNAUTHORIZED")
+  @DisplayName("When authorization is missing, DELETE /subscriptions responds NO CONTENT")
```

Comment:
The test name and respond code don't fit. 
Why was the response changed to "no Content". 
I'd expect a 401 or 403 API response if the client is unauthorized?

Reply:
Also the post and delete requests should respond with the same api respond if the client is unauthorized

Reply:
Hi Steffen, the POST endpoint for token generation should require basic auth. However, the subscription cancellation via token does not (as the possession of the token itself is enough)